ProGetAutomation/Functions/Remove-ProGetAsset.ps1

```diff
@@ -0,0 +1,41 @@
+function Remove-ProGetAsset
+{
+    <#
+        .SYNOPSIS
+        Used to remove assets from the proget asset manager. 
+
+        .DESCRIPTION
+        This function removes assets from the proget asset manager. A session, assetName and assetDirectory is required. 

```

Comment:
Change `This function` to `The \`Remove-ProGetAsset\` function`.

Fix the case of `proget` to `ProGet`.

Reply:
Done, Did this for Add and Get as well